libs/sdk-core/src/lib/db/resource.ts

```diff
@@ -155,6 +157,55 @@ export class Resource implements IResource {
     );
   }
 
+  setLabels(
+    fieldId: string,
+    fieldType: string,
+    paragraphId: string,
+    labels: Classification[],
+    preserve = true,
+  ): Observable<void> {
+    const fieldmetadata = preserve && this.fieldmetadata ? [...this.fieldmetadata] : [];
+    fieldmetadata.push({
```

Comment:
I have a doubt with this way of preserving the fieldmetadata: when preserve is true, if you call `setLabels` or `setEntities` several times, you will push a new fieldMetadata for labels instead of replacing the previous one as it was done before in `setLabels` from api.ts